src/coroutines/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+list(APPEND YACLIB_INCLUDES
+  ${YACLIB_INCLUDE_DIR}/coroutines/standalone_coroutine.hpp
+  ${YACLIB_INCLUDE_DIR}/coroutines/context/stack.hpp
+  ${YACLIB_INCLUDE_DIR}/coroutines/context/stack_allocator.hpp
+  ${YACLIB_INCLUDE_DIR}/coroutines/context/stack_view.hpp
+  ${YACLIB_INCLUDE_DIR}/coroutines/context/default_allocator.hpp
+  )
+
+list(APPEND YACLIB_HEADERS
+  # todo: pick based on a system
```

We write todo like this:
```suggestion
  # TODO(myannyax) Pick based on a system
```